tests/models/pdf/test_raw_pdf_reimport.py

```
@@ -0,0 +1,370 @@
+
+        # Test uuid property
+        assert raw_pdf.uuid == raw_pdf.pdf_hash, (
+            "uuid property should return pdf_hash for backward compatibility"
+        )
+
+        assert raw_pdf.uuid == expected_hash, (
+            "uuid property should match the calculated hash"
+        )
+
+    def test_get_raw_file_path_finds_file_via_file_field(
+        self, center, sample_pdf_content, tmp_path
```

Ruff (0.14.3)

Unused method argument: `tmp_path`

(ARG002)